sql/solutions/lesson6.md

```diff
@@ -0,0 +1,93 @@
+---
+layout: page
+title: Lesson 6 solutions to exercises
+---
+# Lesson 6 solutions to exercises
+
+## L6.1 List all the hosts that had at least one workshop hosted.
+
+```SQL
+SELECT h.name
+  FROM hosts h
+ WHERE EXISTS (
+          SELECT *
+            FROM workshops w
+           WHERE h.id = w.host_id
+         );
+```
+## L6.2 List all the persons that have someone else with their same age (Use only JOINs)
+
+```SQL
+SELECT DISTINCT p1.name, p1.age
+  FROM people p1
+       INNER join people p2
+       ON p1.age = p2.age
+ WHERE p1.name <> p2.name;
```

Comment:
> p1.name <> p2.name

Given that the table people has and id column, it's always better to use that column for the purpose of excluding comparing a row with itself.
Something like

> p1.id <> p2.id;